code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -575,16 +485,16 @@
 	id = "borg_upgrade_rename"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/rename
-	materials = list(MAT_METAL=35000)
-	construction_time = 120
+	materials = list(MAT_METAL = 5000)
```

Comment:
borg upgrades changes here

Reply:
yeah but see borg upgrade cost changes shouldnt be in a PR titled "new mech weapons" in the first place